src/style.css

```diff
@@ -1,3 +1,78 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@300&family=Roboto:wght@400;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
```

Comment:
```suggestion
```
You have already applied it on line 4

Reply:
not fixed https://github.com/mate-academy/layout_moyo-header/pull/4104/files#r1393204350